### PR TITLE
Load OPA Policy by kube-mgmt

### PR DESCRIPTION
**Pull request by hiyosi**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

----

In previously PR(yahoo/k8s-athenz-identity/pull/18), I submitted to make sure OPA read the Policy.
However, in this PR, I'll revert the previous method to load policy by kube-mgmt, according to the kube-mgmt's README. This is more k8s way and load policy dynamically.
Also eliminates the following RBAC errors in mgmt container.

```
github.com/open-policy-agent/kube-mgmt/pkg/configmap/configmap.go:159: Failed to list *v1.ConfigMap: unknown (get configmaps)
```

### CODE REVIEW

**Comment by hiyosi** [comment]
What do you think which way is better? #18 or [This](#20)

File: k8s/identityd.yaml [hunk 1]
```diff
@@ -21,13 +21,10 @@ spec:
...
         imagePullPolicy: IfNotPresent
         name: opa
-        volumeMounts:
-        - mountPath: /config
-          name: config
-          readOnly: true
       - args:
         - --opa-url=http://127.0.0.1:8181/v1
         - --enable-policies=true
+        - --policies=default
```
Comment (prabushyam) [inline k8s/identityd.yaml]: This approach is definitely better. We have to double check with the `sys.authz` package rego (where we will move the current identity rego policies) as OPA expects them to be available at process startup.

File: k8s/rbac.yaml [hunk 2]
```diff
@@ -11,6 +11,16 @@ rules:
...
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+  - update
```
[the remark this answers is not in the document]
Comment (hiyosi) [reply k8s/rbac.yaml]: In the kube-mgmt repo, they specify 'update` in the manifest.
https://github.com/open-policy-agent/kube-mgmt/blob/master/manifests/deployment.yml#L25

I'll check on my environment without `update` verb.

File: k8s/identityd.yaml (hunk 1 above)
Comment (hiyosi) [reply k8s/identityd.yaml]: I think this problem is same as yahoo/k8s-athenz-identity/pull/23.
The next step, I want to be able to specify the namespace as parameters when deploying, using such as kustomize.

File: k8s/rbac.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (hiyosi) [reply k8s/rbac.yaml]: I checked on my env,  it seemed not to need `update`

[the remark this answers is not in the document]
Comment (hiyosi) [reply k8s/rbac.yaml]: Removed 3f7e4b5

File: k8s/identityd.yaml (hunk 1 above)
Comment (mcieplak) [reply k8s/identityd.yaml:27]: This is something we could make configurable through the helm template https://github.com/yahoo/k8s-athenz-identity/pull/14
Like @prabushyam mentioned, we need to see if this approach will be able to work with the `sys.authz` package as well.

**Pull request closed without merging**